src/test/scala/org/broadinstitute/dsde/workbench/sam/api/ResourceRoutesV1Spec.scala

```diff
@@ -30,40 +31,6 @@ class ResourceRoutesV1Spec extends FlatSpec with Matchers with ScalatestRouteTes
 
   private val managedGroupResourceType = configResourceTypes.getOrElse(ResourceTypeName("managed-group"), throw new Error("Failed to load managed-group resource type from reference.conf"))
 
-  private object SamResourceActionPatterns {
```

Comment:
changes in `TestSamRoutes`, `ResourceRoutesV1Spec` and `ResourceRoutesV2Spec` are due to conditional creation of admin resource types. Before this change the tests themselves were responsible for calling `initResourceTypes` but the way it was setup the resource types already existed so the admin resources would not get created. This change moves all the admin resource type stuff and init to `TestSamRoutes` which is probably more true to life anyway.